CHANGES.md

```diff
@@ -70,6 +70,7 @@
 ## I/Os
 
 * Support for X source added (Java/Python) ([#X](https://github.com/apache/beam/issues/X)).
+* Upgraded Iceberg dependency to 1.9.2 ([#35981](https://github.com/apache/beam/pull/35981))
```

Comment:
We probably want to note in "Breaking change" that  expansion services dropped Java 8 support (e.g. Python pipeline uses cross-language Java transform need to have Java11+)

Reply:
Good call, done